docs/developer/address.mdx

```diff
@@ -177,3 +177,18 @@ Each field is validated separately, and each issue has a corresponding error cod
 
 - `postalCode` with the wrong format throws the `INVALID` error
 - missing fields mentioned in `requiredFields` throw the `REQUIRED` error
+
+## Skipping the address validation
+
+Since version 3.19, Saleor may completely skip the address validation. To do so, the `AddressInput` must receive `skipValidation` flag set to `true`.
+This will turn off whole validation, except `country` field - this one requires proper `CountryCode` enum.
+
+Please note, that to use `skipValidation` flag, Saleor may demand some extra permissions. Also, not all mutations support this feature.
+
+- `checkoutCreate` - requires `HANDLE_CHECKOUT` and `AUTHENTICATED_APP` permissions.
+- `accountAddressCreate`, `accountAddressUpdate` and `accountUpdate` do not support the feature.
+
+:::warning
+When providing invalid address, Saleor may be unable to complete some external requests.
```

Comment:
Providing an invalid address may cause Saleor to be unable to complete some external requests. For example, an incorrect address could result in errors in the tax app response and an inability to create an order.